q2_longitudinal/_longitudinal.py

```diff
@@ -418,9 +418,8 @@ def plot_feature_volatility(output_dir: str,
         _validate_metadata_is_superset(importances, table.T)
 
     # filter table, importances based on importance threshold / feature count
-    if importance_threshold is not None or feature_count is not None:
```

Comment:
@nbokulich, i dropped the guard here, since feature_count can never be `None`, and more specifically, the `_importance_filtering` util is built in a way that can allow it to no-op if necessary, makes sense to just pass on through.